src/Microsoft.TemplateEngine.Edge/Template/ITemplateMatchInfo.cs

```diff
@@ -1,9 +1,11 @@
+using System;
 using System.Collections.Generic;
 using Microsoft.TemplateEngine.Abstractions;
 
 namespace Microsoft.TemplateEngine.Edge.Template
 {
     // Replacement for IFilteredTemplateInfo
+    [Obsolete("moved to " + nameof(Abstractions.TemplateFiltering) + " namespace" )]
```

Comment:
When do we plan to remove these? .NET 7?
Nit: First letter capital.

Reply:
I think better way to do it will be wait until .NET 7, however we can do it earlier if everyone agrees to. We already made some changes to things mark obsolete and other public members in this release, so I'm not sure of benefit.